adobe/react-spectrum
# ListView drop indicator fixes

- Fixes roles for root drop indicator
- Adds hidden button for dropping on an item
- Fixes to not render drop indicators if not in drag session
- Fixes insertion indicator z-index to be above row border when rows are selected

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

Test ListView DnD stories for desired behavior/announcements for dropping between, on, and on root.

To test insertion indicator z-index fix:
In 'Reorder' story, select some rows, then drag an item over the selected rows. Insertion indicator should be above the border of the selected rows. 

## 🧢 Your Project:

RSP
